client/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "rock-with-me",
```

Comment:
Rename the `"name"` to `"PROJECT-client"`

Reply:
thank you very much!